src/frontend/components/DowloadVideo/DownloadVideo.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box } from 'grommet';
+import * as React from 'react';
+import { defineMessages, FormattedMessage, Messages } from 'react-intl';
+
+import { Video, videoSize } from '../../types/tracks';
+
+const messages: Messages = defineMessages({
+  downloadVideo: {
+    defaultMessage: 'Download video with quality: ',
+    description: 'Text to download the video',
+    id: 'components.DownloadVideo.downloadVideo',
+  },
+  480: {
+    defaultMessage: 'Mobile',
+    description: 'Mobile quality video',
+    id: 'components.DownloadVideo.480',
+  },
+  720: {
+    defaultMessage: 'Normal',
+    description: 'Normal quality video',
+    id: 'components.DownloadVideo.720',
+  },
+  1080: {
+    defaultMessage: 'Hight',
+    description: 'Hight quality video',
+    id: 'components.DownloadVideo.1080',
+  },
+});
+
+export const DownloadVideo = ({ video }: { video: Video }) => {
+  const elements: JSX.Element[] = [];
+  return (
+    <Box align="center" justify="start" direction="row" pad="small">
+      <FormattedMessage {...messages.downloadVideo} />
+      &nbsp;
+      {(['1080', '720', '480'] as videoSize[])
+        .reduce((acc: JSX.Element[], size: videoSize) => {
+          acc.push(
+            <a
+              key={size}
+              target="_blank"
+              rel="noopener noreferrer"
```

Comment:
This opens a new tab with the video that the browser starts to play. I've tried to add the `download` attribute without success. IMO our best option is to add a message inviting the user to right-click on the links and choose to "save as".

Or is there a chance that we can modify cloudfront response headers conditionally to add:

```Content-Disposition: Attachment;filename=FILENAME_HERE```

Reply:
I will take a look but if we do that I don;t know if the player will still continue to play the video.

Reply:
It looks like not possible to change the content-type. The content-type used is the one set when the video is uploaded to `s3` bucket.

Reply:
Would it be possible maybe to open a specific CloudFront distribution with this `content-type`?

How does it work on videofront?

Reply:
> How does it work on videofront?

It opens a new tab with the video inside ¯\_(ツ)_/¯

Reply:
After digging in the AWS documentation and trying to modify some headers here is the situation:

- It is possible to force the `Content-Type` metadata in the source bucket when you upload it.
- modifying the `Content-Type` is IMO dangerous because every type of files will be now uploadable. It is still possible to check the type before starting the update but...
- If you modify it, `MediaConverter` will save the file in the destination bucket and there is no option to set the `Content-Type` as you can see [here](https://docs.aws.amazon.com/mediaconvert/latest/apireference/jobs.html#jobs-model-filegroupsettings) so the file converted will be again with the `video` `Content-Type`.

Possible solution:

Create a new lambda responsible to change the metadata `Content-Type` and set it to `binary/octet-stream` on files created in the destination bucket.

Reply:
I think we must declare an issue and deal with this improvement later. WDYT?

Reply:
I'm already working on it :smile:

Reply:
Ok then.